dolphinscheduler-task-plugin/dolphinscheduler-task-sql/src/main/java/org/apache/dolphinscheduler/plugin/task/sql/SqlTask.java

```diff
@@ -133,8 +140,11 @@ public void handle() throws TaskException {
                     DbType.valueOf(sqlParameters.getType()),
                     sqlTaskExecutionContext.getConnectionParams());
 
+            if (DbType.valueOf(sqlParameters.getType()).isSupportMultipleStatement()) {
+                sqlSeparator = "";
```

Comment:
We could have something like `private static final String HIVE_SQL_SEPARATOR = ""` and put it with other separators on top of the class for better code maintainability.